modules/videoio/src/cap_ffmpeg_impl.hpp

```diff
@@ -1580,6 +1617,10 @@ double CvCapture_FFMPEG::getProperty( int property_id ) const
         if (rawMode)
             return -1;
         break;
+    case CAP_PROP_LF_KEY_FRAME:
+        return packet.flags == 1;
```

Comment:
Wrong usage.

> 1

Don't use magic number.

> == 1

Must be `((packet.flags & AV_PKT_FLAG_KEY) != 0) ? 1 : 0`

Check `libavcodec/packet.h` for other values.

---

Anyway this value is not about the frame. This property is about **the packet** (due to B/P frames). Frames may be returned in a different order.